src/components/CurrentImage/index.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
+import { Col, Row } from "antd";
+import puzzledImage from "./../../images/habitat.jpg";
+import "./index.css";
+
+// CurrentImage is the component that generates and includes the tiles of the puzzle
+// it generates 6 tiles ordered vertically on top of each other
+// each tile's background is 1/6 of the image. After the tiles are generated they are put
+// in an array and ordered randomly
+// a user can click on one tile to select and then click on another tile to swap their places
+// the user will swap the places of each tile to put it in its correct position
+// when all tiles are placed in their correct position the full image is displayed
+const CurrentImage = () => {
```

Comment:
this component should be called PuzzleGrid or PuzzleTileGrid or something like this

CurrentImage still seems to me like its the "final image" that they wanna do, not the actual thing that they are editing